### PR TITLE
fixed styling for PDP page

### PR DESCRIPTION
**Pull request by smruti-kshirsagar**

### CODE REVIEW

**Comment by rahimoddin** [comment]
Please resolve the conflicts.
Do not commit anything to products.json file.
Pull the latest changes from develop and verify it locally.

**Comment by smruti-kshirsagar** [comment]
Hi Rahim,
I have resolved the conflicts, and due to latest update PDP page was broken so I fixed it as well in this PR.
I got latest products.json file from -> https://github.com/rahimoddin/shopping-site/blob/develop/public/static/data/products.json

File: src/views/Product.vue [hunk 1]
```diff
@@ -67,14 +85,13 @@
     </section>
     <section class="suggestion-collection mt-5 mb-5">
       <h4 class="text-center">You may also like</h4>
-      <div class="product-listing col-12 col-md-10 row">
-        <!-- <ProductTile v-for="product in products" :key="product.sku" :product="product"/> -->
+      <div class="product-listing col-12 col-md-10 row mt-3">
+        <!-- <ProductTile
```
Comment (rahimoddin) [inline src/views/Product.vue:89]: Please remove this commented block if not needed.

File: src/views/Product.vue [hunk 2]
```diff
@@ -88,6 +105,7 @@ import ProductGallery from "@/components/product/ProductGallery.vue";
 import ProductReviews from "@/components/product/ProductReviews.vue";
 import ProductColor from "@/components/product/ProductColor.vue";
 import ProductSize from "@/components/product/ProductSize.vue";
+// import ProductTile from "@/components/plp/ProductTile.vue";
```
Comment (rahimoddin) [inline src/views/Product.vue:108]: Please remove commented block

File: src/views/Product.vue [hunk 3]
```diff
@@ -98,29 +116,34 @@ export default {
...
   },
   computed: {
     ...mapGetters('product', {
       product: 'getCurrentProduct',
     }),
     checkWishlist() {
       return this.$store.getters['myaccount/getItems']
-    }
+    },
+    ...mapGetters('category', {
```
Comment (rahimoddin) [inline src/views/Product.vue:129]: Where 'products' being used?. I think it can be removed.

File: src/views/Product.vue [hunk 4]
```diff
@@ -157,22 +180,39 @@ export default {
         })
       }
     }
+  },
+  created() {
+    this.$store.dispatch("category/getProductsByCategory");
```
Comment (rahimoddin) [inline src/views/Product.vue:185]: why getProductsByCategory on product page?

**Pull request merged**